2-mandatory/3-financial-times.js

```diff
@@ -24,6 +37,14 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    let numArticle = [];
+    let pattern =/[0-9]/g;
+    for (let i=0; i<allArticleTitles.length; i++){
+        if (allArticleTitles[i].search(pattern)!= -1 ){
```

Comment:
Better to prefer `!==` over `!=` because `!==` gives more predictable results (`!=` will try to convert the types). Have a look for 'equality operator and type coercion' if interested but don't worry right now if not.